5calls/app/src/main/java/org/a5calls/android/a5calls/model/DatabaseHelper.java

```diff
@@ -76,10 +73,26 @@ public void onCreate(SQLiteDatabase db) {
 
     @Override
     public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
-        if (newVersion >= 2) {
+        if (oldVersion < 2) {
```

Comment:
newVersion is supplied & can be used to make sure we don't upgrade too far. I like to make these step-wise, making sure not to go past the supplied newVersion:

if (oldVersion == 1 && oldVersion < newVersion) {
 // do upgrade from 1 to 2
 oldVersion = 2;
}
if (oldVersion == 2 && oldVersion < newVersion) {
  // do upgrade from 2 to 3
  oldVersion = 3;
}

Reply:
Done.